libraries/bezier-rs/src/bezier/transform.rs

```diff
@@ -1,11 +1,39 @@
 use super::*;
+
+use crate::compare::compare_points;
 use crate::utils::{f64_compare, TValue};
+use crate::{AppendType, ManipulatorGroup, Subpath};
 
 use glam::DMat2;
 use std::f64::consts::PI;
 
 /// Functionality that transform Beziers, such as split, reduce, offset, etc.
 impl Bezier {
+	/// Returns a linear approximation of the given [Bezier]. For higher order [Bezier] this means simply dropping the handles.
+	pub fn to_linear(&self) -> Bezier {
+		Bezier::from_linear_dvec2(self.start(), self.end())
+	}
+
+	/// Returns a quadratic approximation of the given [Bezier].
+	pub fn to_quadratic(&self) -> Bezier {
+		let handle = match self.handles {
+			BezierHandles::Linear => self.start,
+			BezierHandles::Quadratic { handle } => handle,
+			BezierHandles::Cubic { handle_start, handle_end } => (handle_start + handle_end) / 2.,
+		};
+		Bezier::from_quadratic_dvec2(self.start, handle, self.end)
+	}
+
+	/// Returns a cubic approximation of the given [Bezier].
+	pub fn to_cubic(&self) -> Bezier {
+		let (handle_start, handle_end) = match self.handles {
+			BezierHandles::Linear => (self.start, self.end),
+			BezierHandles::Quadratic { handle } => (self.start + (2. / 3.) * (handle - self.start), self.end + (2. / 3.) * (handle - self.end)),
```

Comment:
Comment this reference for the math: https://stackoverflow.com/a/63059651/775283